Dockerfile

```diff
@@ -25,8 +26,9 @@ RUN apt-get -y update && apt -y install \
   git-core \
   libboost-program-options-dev
 
-RUN git clone -b ${OSM2PGSQL_TAG} --single-branch https://github.com/openstreetmap/osm2pgsql.git ./osm2pgsql && \
+RUN git clone ${OSM2PGSQL_REPOSITORY} ./osm2pgsql && \
```

Comment:
maybe we can still use --single-branch as we only want the master.

Reply:
as discussed it will remain without the `--single-branch` flag